bxml/buildingBlocks/createCall.md

```diff
@@ -0,0 +1,23 @@
+{% method %}
+# Create an Outbound Call
+
+## Concept
+
+To create an outbound call, we will use the RestAPI. The biggest difference is the callbackHttpMethod must be <code class="get">GET</code>. For more information about creating an outbound call, visit the [Full API Reference](../../methods/calls/postCalls.html)
+
+## Code
+
+The from number is a Bandwith number you own. Change the `to`, `from`, and `callbackUrl` parameters.
+{% common %}
```

Comment:
Give this example a title, something like `### Making an outbound call to use with BXML`